io.github.chidiwilliams.Buzz.yml

```diff
@@ -0,0 +1,51 @@
+app-id: io.github.chidiwilliams.Buzz
+runtime: org.freedesktop.Platform
+runtime-version: '24.08'
+sdk: org.freedesktop.Sdk
+command: run-buzz.sh
+finish-args:
+  - --socket=wayland
+  - --socket=fallback-x11
+  - --socket=pulseaudio
+  - --talk-name=org.freedesktop.secrets
```

Comment:
`=org.freedesktop.secrets` why is this needed?

Reply:
It is used to access the `keyring` functionality and store users API keys for the API server if user adds configuration for the remote transcription API server that requires the API key.

Reply:
How about using the secrets portal https://flatpak.github.io/xdg-desktop-portal/docs/doc-org.freedesktop.portal.Secret.html ? Then the permission is not needed.

Reply:
I tested the app without this permission and it did not work, so the python `keyring` module does not handle secrets portals as is. Happy to implement this in the next version of the app as it will be also needed for the Snap version.